### ISSUE TITLE
Updated Shot List for Website

### ISSUE THREAD

**Issue by sean8300** [sean8300/listerassister_issues#395]
Please Replace the Shotlist on the website with this one:   https://1drv.ms/b/s!AqLylZLphekuj7sIfQoFF4dTB3af-Q?e=DB6Rj3 
 Submitted By:  Nelson Tello

**Comment by sean8300** [comment]
Believe this is already done, assigning Juvi so he can learn where this is/how it works

**Comment by sean8300** [comment]
Updated front end of the site, removed the version tag from the file, will now just use the "StandardShotList.pdf" as it should.  Waiting on backend changes
